core/src/avm1/globals/bitmap_data.rs

```diff
@@ -1205,38 +1205,35 @@ pub fn load_bitmap<'gc>(
 
     let movie = activation.target_clip_or_root().movie();
 
-    let renderer = &mut activation.context.renderer;
-
     let character = movie
         .and_then(|m| library.library_for_movie(m))
         .and_then(|l| l.character_by_export_name(name));
 
-    if let Some(Character::Bitmap(bitmap_object)) = character {
-        if let Some(bitmap_handle) = bitmap_object.bitmap_handle() {
-            if let Some(bitmap) = renderer.get_bitmap_pixels(bitmap_handle) {
-                let new_bitmap_data = BitmapDataObject::empty_object(
-                    activation.context.gc_context,
-                    Some(activation.context.avm1.prototypes().bitmap_data),
-                );
+    if let Some(Character::Bitmap {
+        bitmap: bitmap_object,
+        initial_data,
```

Comment:
This changes the behavior - we'll now use the original data each time we create an instance of a `BitmapData` subclass, rather than getting the current contents of the original `Bitmap` instance. I'm pretty sure that this is the correct behavior (since each `BitmapData` instance should be independent), but I need to test this.